analyzer/server.go

```diff
@@ -121,27 +125,43 @@ func (s *Server) createStartupCapture(ch *api.CaptureAPIHandler) error {
 	logging.GetLogger().Infof("Invoke capturing of type '%s' from startup with gremlin: %s and BPF: %s", captureType, gremlin, bpf)
 	capture := types.NewCapture(gremlin, bpf)
 	capture.Type = captureType
-	return ch.Create(capture, nil)
+	return apiHandler.Create(capture, nil)
 }
 
 // Start the analyzer server
 func (s *Server) Start() error {
-	if s.storage != nil {
-		s.storage.Start()
+	if err := s.httpServer.Listen(); err != nil {
+		return err
 	}
 
-	if err := s.httpServer.Listen(); err != nil {
+	if err := s.hub.Start(); err != nil {
+		return err
+	}
+
+	s.etcdClient.Start()
+
+	if s.graphStorage != nil {
```

Comment:
I wonder if it is not better to start storage before http.Listen like before in order to be sure that we won't have pod -> hub message while not having the storage ready ?